README.md

```diff
@@ -1579,6 +1579,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Rajan Kumar Sharma](https://github.com/SharmaRajan)
   - [Rajat Mehra](https://github.com/rajat-mehra05)
   - [Raju Ghorai](https://github.com/coderj001)
+  - [Rajnish](https://github.com/Rajnish17)
```

Comment:
Your name should be above Raju Ghorai.
Please update accordingly. Any help, let us know. 🙂